### ISSUE TITLE
Tail-Logs script

### ISSUE THREAD

**Issue by graste** [honeybee/honeybee-agavi-cmf-vendor#2]
As we want to switch away from the `Makefile` we might add bash scripts or put more scripts into the ```composer.json``` `scripts` section. I'd suggest PHP scripts that are callable directly in the `bin/` folder instead of composer scripts for everything as that might be easier to debug for certain easy use-cases. For the `Makefile::tail-logs` I suggest a ```bin/tail-logs``` bash script:

```
#!/usr/bin/env bash

PHP_ERROR_LOG=$(php -i | grep -P '^error_log' | cut -f '3' -d " ")

tail -f "${PHP_ERROR_LOG}" app/log/*.log
```

**Comment by MrHash** [comment]
Makes sense but might be a compatibility issue when running on different platforms? Most likely OSX compatibility should be easy enough to ensure.. but Windows?
